registry/repository.go

```diff
@@ -98,25 +98,16 @@ func (c *RepositoryClient) PutSignatureManifest(ctx context.Context, signature [
 		return notation.Descriptor{}, SignatureManifest{}, err
 	}
 
-	// generate artifact manifest
-	artifactManifest := artifactspec.Manifest{
-		MediaType:    artifactspec.MediaTypeArtifactManifest,
-		ArtifactType: ArtifactTypeNotation,
-		Blobs:        []artifactspec.Descriptor{signatureDesc},
-		Subject:      artifactDescriptorFromNotation(manifest),
-		Annotations:  annotaions,
+	manifestDesc, err := c.uploadSignatureManifest(ctx, artifactDescriptorFromNotation(manifest), signatureDesc, annotaions)
```

Comment:
nit, `annotaions`.

Reply:
oops, thanks for catching!

Reply:
Can we rename `manifest` to `subjectManifest` to make it clearer, we are dealing with signature blob, subject and signature manifest descriptors here. Same for line 153.

Reply:
that makes sense, already updated.